### PR TITLE
Edit DOB SSN cucumber refactor

### PR DESCRIPTION
**Pull request by quisbertdiana**
Refactor edit dob snn cucumber, added new cucumber validating email/username already in use.

### CODE REVIEW

File: features/step_definitions/edit_dob_ssn_steps.rb [hunk 1]
```diff
@@ -25,14 +25,27 @@
...
+When(/^Admin will see the Edit DOB SSN button$/) do
+  find_link('Edit DOB / SSN')
+end
+
+When(/^Admin clicks Action for a person on families index page$/) do
+  find('.dropdown.pull-right', text: 'Actions').click
+end
+
+When(%r{^Admin clicks on edit DOB/SSN link$}) do
+  click_link('Edit DOB / SSN')
```
Comment (markgoho) [inline features/step_definitions/edit_dob_ssn_steps.rb:46]: hard-coded english needs to be removed

File: features/step_definitions/individual_steps.rb [hunk 2]
```diff
@@ -1073,3 +1073,27 @@ class VlpDocument
   fill_in "user[password]", :with => "aA1!aA1!aA1!"
   find('.sign-in-btn').click
 end
+
+Then(/Individual creates a new HBX account via username$/) do
+  sleep 5
```
Comment (markgoho) [inline features/step_definitions/individual_steps.rb:1078]: why are we using `sleep` here?

**Review by markgoho — DISMISSED** [review]

File: features/step_definitions/edit_dob_ssn_steps.rb (hunk 1 above)
Comment (quisbertdiana) [reply features/step_definitions/edit_dob_ssn_steps.rb:46]: will make changes to remove hard coded english

**Comment by markgoho** [comment]
@quisbertdiana it's great to see this work put in to improving our cucumbers!

ideally, we want pull requests to represent a change to one thing -- this PR looks like it's doing two things:
1. refactoring a cucumber related to dob/ssn
2. adding a new cucumber related to individual account creation

in the future, we may request that a pull request like this that is making two (or more) separate changes should be broken up into separate pull requests

consider the scope of the change that you intend to make and if you need help drawing the boundary between a single change and multiple changes, please don't hesitate to reach out!

**Review by bill-transue — APPROVED** [review]
Refactors edit DOB/SSN feature and adds individual account creation feature using page object models. Good work @quisbertdiana.

**Pull request merged**